azure-iot-device/azure/iot/device/constant.py

```diff
@@ -0,0 +1,12 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""This module defines constants for use across the azure-iot-device package
```

Comment:
I considered naming this module `version.py` instead, but technically the User-Agent isn't really a "version", so it felt weird.

Reply:
what about naming it `client_verison.py` 

---
In reply to: [307432156](https://github.com/Azure/azure-iot-sdk-python-preview/pull/147#discussion_r307432156) [](ancestors = 307432156)

Reply:
User-Agent still realy isn't a "version"

---
In reply to: [307824566](https://github.com/Azure/azure-iot-sdk-python-preview/pull/147#discussion_r307824566) [](ancestors = 307824566,307432156)